plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -78,14 +76,7 @@ export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
     }
 
     setLoading(true);
-    const workflowCategory = getWorkflowCategory(
-      schemaResponse?.workflowItem.definition,
-    );
-    if (workflowCategory === WorkflowCategory.ASSESSMENT) {
-      Object.assign(parameters, { businessKey: crypto.randomUUID() });
-    } else {
-      Object.assign(parameters, { businessKey: businessKey ?? '' });
-    }
+    Object.assign(parameters, { businessKey: businessKey ?? '' });
```

Comment:
What if we assign the `businessKey` only if it is not `undefined`?
I see no value for `businessKey=<empty_string>`. It could generate noise IMO.